docs/guidelines/marketing_guide_to_firefox_l10n.md

```diff
@@ -0,0 +1,164 @@
+
+Firefox as a _brand_ combines the Firefox ethos and the expanding range of products.
+
+(Mozilla is the not-for-profit organization behind Firefox.)
+
+**Brand promise**:
+
+Firefox fights for you
+
+**What are we fighting for**:
+
+Firefox fights for our users by creating products that give people control and agency over their online lives.
+
+**Firefox Products**:
+
+* Firefox Quantum (Desktop browser)
+* Firefox Mobile (für iOS und Android)
+* [Firefox Send](https://send.firefox.com/)
+* [Firefox Monitor](https://monitor.firefox.com/)
+* [Firefox Lockbox](https://lockbox.firefox.com/)
+* [Pocket](https://play.google.com/store/apps/)
+* [Firefox Reality](https://mixedreality.mozilla.org/firefox-reality/)(VR browser)
+
+## Personality and tonality
+
+### Who are we talking to?
+
+We describe our target audience as Conscious Choosers. This refers to people who make or want to make conscious decisions, online and offline.
+
+Conscious Choosers:
+
+* Develop preferences and make consumer choices carefully and consciously
+* Strive to reconcile their actions with their values and ideals
+* Are demanding customers
+* Often do their own research to fully understand all options
+
+### Our Brand Attributes and the Firefox Brand Personality
+
+| **Attribute** | **Firefox Brand Personality** |
+| --- | --- |
+| Opinionated | **Firefox acts out of conviction and takes clear positions with confidence.**  <br><br>Our products show that our motivation derives from our brand vision. Our brand ethos is at the root of everything we do. And that&#39;s what we want to communicate to our users and partners. |
+| Open | **Firefox thinks that the internet should be open, accessible and safe for everyone.**  <br><br>We strive for open conversation and collaboration. We are: Open minded. Open-hearted. Open source. A global perspective is an integral part of our brand. We speak many languages and try to take different perspectives. |
+| Kind | **Firefox anticipates needs, offering solutions and alternatives where users&#39; freedom and rights are threatened or abused.**  <br><br>We want the best for our users and the world, so we lead by example. We develop better products, we start dialogues, we work openly and with others, we educate ourselves and others, and we inform our users. In doing so, we act empathically toward all people. |
+| Radical | **Firefox questions the status quo and common practices of tech giants and struggles with confidence for a better internet.**  <br><br>Looking optimistically into the future of the internet is a radical act. Bringing users&#39; needs before our own is just as radical. We challenge the status quo because we think it&#39;s the right thing to do. |
+
+The Firefox Brand Personality is defined by the interaction of these attributes. Depending on the context, an attribute may be more forward or less forward.
+
+## Tone
+
+Our tone of voice and language choices are:
+
+| Tone | |
```

Comment:
The second column should have a text

Reply:
Actually this is something I added. Original document has no heading row, which looks strange for a table.